docs/ref/contrib/gis/geoquerysets.txt

```diff
@@ -523,7 +523,8 @@ With PostGIS, on every distance lookup but :lookup:`dwithin`, an optional
 third element, ``'spheroid'``, may be included to tell GeoDjango
 to use the more accurate spheroid distance calculation functions on
 fields with a geodetic coordinate system (e.g., ``ST_Distance_Spheroid``
-would be used instead of ``ST_Distance_Sphere``).
+would be used instead of ``ST_Distance_Sphere``). Still on PostGIS, with
```

Comment:
I think we could omit the "Still on PostGIS" since the entire paragraph seems to be about PostGIS but in case you'd like to keep it, a less awkward phrasing would be "Also on PostGIS". The current version is okay, but we could possibly simplify the sentence like so: "The simpler ``ST_Distance`` function is used with projected coordinate systems." if you like. Anyway, fine for me if it clarifies the issue for the ticket reporter.